README.md

```diff
@@ -40,16 +40,16 @@ Please follow the [ESPHome documentation](https://esphome.io/guides/getting_star
 
 ### Pin-out
 
-| RJ45 pin | Name      | ESP32        |
-| -------- | --------- | ------------ |
-| 8        | +5V (VDD) | VIN          |
-| 7        | GND       | GND          |
-| 6        | TX        | TX2 (GPIO17) |
-| 5        | RX        | RX2 (GPIO16) |
-| 4        | PIN 20    | D23 (GPIO23) |
-| 3        | (unknown) |              |
-| 2        | SWIM      |              |
-| 1        | RES       |              |
+| RJ45 pin | Name      | ESP32        | ESP32-C3     |
+| -------- | --------- | ------------ | ------------ |
+| 8 🟫     | +5V (VDD) | VIN          | VIN          |
+| 7 ⬜🟫   | GND       | GND          | GND          |
+| 6 🟩     | TX        | TX2 (GPIO17) | TX2 (GPIO21) |
```

Comment:
Do TX2, RX2, D23 makes sense for ESP32-C3?

Reply:
Update